server/modules/playlist_loader.js

```diff
@@ -25,61 +25,54 @@ var Layout = require('server/modules/layout');
 var ModuleDef = require('server/modules/module_def');
 var library = require('server/modules/module_library');
 
+function parseJson(jsonString) {
+  return RJSON.parse(jsonString);
```

Comment:
WHY?

JSON is so readable...

/me sobs

Reply:
Are you saying why not just straight JSON? Because it's too machine-generated to be easy to write: you can't have a trailing comma in a list, for example, and this happened to us so often in the beginning that we switched parsers.

Reply:
/me recoils in horror

I had no idea that I was off the standard with my trailing commas.